python/exercise2/time_window.py

```diff
@@ -20,12 +20,20 @@ def __init__(self, start_time, end_time, delta):
                 
         """
         
+        # Error checking of delta input
```

Comment:
it would be good to add documentation indicating why these numbers were added for the conditions